src/lib/controller/index.ts

```diff
@@ -235,14 +235,14 @@ export class Controller {
         responseList.map(res => {
           this.#afterHook && this.#afterHook(res);
           // Log the response
-          Log.debug({ ...reqMetadata, res }, "Check response");
+          Log.info({ ...reqMetadata, res }, "Check response");
         });
 
         let kubeAdmissionResponse: ValidateResponse[] | MutateResponse | ResponseItem;
 
         if (admissionKind === "Mutate") {
           kubeAdmissionResponse = response;
-          Log.debug({ ...reqMetadata, response }, "Outgoing response");
+          Log.info({ ...reqMetadata, response }, "Outgoing response");
```

Comment:
```suggestion
          Log.debug({ ...reqMetadata, response }, "Outgoing response");
```

We don't want to replace all the info with debug. We just want to preserve `npx pepr monitor` when the log level is info.